In [2]:
import os


In [4]:

#root = r'D:\视频录制0715_V1.0\bin_demo\x64\Release\records_1119\Camera1'
#root=r'D:\视频录制0715_V1.0\bin_demo\x64\Release\records_1123_1126\Camera1'
#root=r'D:\视频录制0715_V1.0\bin_demo\x64\Release\records_1130_1201\Camera1'
#root = r'D:\视频录制0715_V1.0\bin_demo\x64\Release\records_1202\Camera1'
root = r'D:\WorkSpace\Datasets\Eye_status_images\close'
vals_name = []
val_list = os.listdir(root)
for file in val_list:
    val_name = file.split('Person_')[-1].split('_')[0]
    vals_name.append(val_name)

In [5]:
vals_name_sets = set(vals_name)
vals_dict = {}
for item in vals_name_sets:
    vals_dict.update({item:vals_name.count(item)})

In [8]:
vals_dict

{'kangfude': 235,
 'liupeng': 251,
 'yuanxiaoqing': 264,
 'lizheng': 256,
 'lidengwu': 285,
 'zhangwenzheng': 292,
 'tiandaipeng': 384,
 'malin': 252,
 'maoling': 314,
 'jiangleqi': 269,
 'fangxiang': 242,
 'denghao': 311,
 'sunluomeng': 355,
 'liuying': 255,
 'yangshen': 298,
 'wangkangpeng': 258,
 'suoyan': 248,
 'zhuqingyun': 347,
 'wangshiyu': 387,
 'wangtao': 232,
 'cailili': 368,
 'licheng': 268,
 'tangyi': 199,
 'zhangwulong': 236,
 'sunyansong': 123,
 'caijinwen': 199,
 'luofangzhou': 108,
 'heenying': 250}

{'gaofeng': 109,     1
 'chenwencan': 162,        1       
 'shenying': 157,1
 'make': 103,           1
 'wangjianwei': 152,     1
 'jiangjing': 154,      1
 'test': 1,
 'huajiahao': 156,      1
 'hejinjian': 148,      1
 'fangyiwei': 149,       1 
 'grab': 1,
 'zhangzhe': 155,      1
 'zhouyong': 155}      1   

{'fengshaohua': 148,       1
 'yanghong': 159,         1
 'xuhaijun': 157,         1
 'xiemengmeng': 151,       1   
 'jinzhonghua': 157,       1
 'xuelei': 157,           1
 'lihaichao': 111,         1
                                'liukangxi': 50,       
 'ludi': 148,            1
 'libin': 153,            1
 'liuzhaoqiang': 153,       1
 'liuzhonggang': 156,       1 
 'wangshi': 155,          1
 'zhouhongyan': 154,        1
                               'zhulili': 106,           1
 'wuyinglan': 152,         1
 'chenjingjing': 156,       1
 'huangjing': 153,         1
                                'wangyinghui': 154,
 'gaojiangjiang': 150,       1
 'grab': 1,
 'zouchen': 153,           1
 'tianpengfei': 154,         1
 'liumingzhang': 157,         1
 'shenwei': 155,            1
 'niuce': 157,             1
 'weiyuanyuan': 157,         1
 'yanghuilong': 129}         1

wangshoupeng': 105,          1
                                        'gusongchao': 106,  1
 'wangxiaoyue': 149,         1
 'xiaqiang': 103,            1
 'luyao': 150,            1
 'zhuhuilin': 149,         1
                                       'fangdongmei': 105,    1
 'zhangwl': 1,
 'grab': 1,
 'liulixiang': 148,           1
                                         'linnan': 104,   1        
 'lulu': 147,               1
                                 'weidong': 105,             1      
 'dengjingheng': 148,          1
 'zhenxiaoyong': 149}          1

'zhudan': 151,           1 
 'luobin': 151,           1
 'guojialu': 154,        1
                                         'xinwei': 108,   1
 'grab': 1,
 'yangchengang': 150,        1
 'zhangqing': 164,         1
 'zhanglan': 157}      1

gengyouqiang': 151,             1
 'xuzhicheng': 152,             1
 'weilinfu': 111,               1 
                                       'hongjun': 111,     1
 'guojiayao': 153,              1
 'grab': 1,
 'lihogli': 1,
 'zhaimengfei': 153,             1
 'lizaichen': 155,              1
                                      'chenjian': 107}     1

In [17]:
print(len(vals_dict))

28


{'gaofeng': 109,      1
 'chenwencan': 162,    1
 'shenying': 157,       1
 'make': 103,        1
 'wangjianwei': 152,      1
 'jiangjing': 154,      1
 'test': 1,
 'huajiahao': 156,      1
 'hejinjian': 148,     1
 'yuhui': 159,        1
 'fangyiwei': 149,    1
 'grab': 1,
 'zhangzhe': 155,      1
 'zhouyong': 155}           1
'housaisai': 148,      1
 'chenjie': 153,      1
 'jiangjingtao': 148,        1
 'xianghanrao': 44,           1
 'songziheng': 153,          1
 'zenghongmei': 155,     1
 'tangchenghua': 151,     1
 'liaoyaowen': 112,         1
 'panzhenghui': 152,          1
 'grab': 1,
 'test': 2,
 'zhangjianke': 152,          1
 'likai': 148,             1
 'zhuyingchun': 92,       1
 'liyanfang': 157,         1
 'qiboaying': 151,            1
 'chenjian': 153,    1
 'chenting': 105,        1
 'liweiyong': 156,          1
 'jiangqingfeng': 113,       1
 'yanghuilong': 154,         1
 'yangjunqian': 148,    1
 'weihongjie': 151,         1
 'zhouhuangkai': 153,       1
 'xianghanyao': 115,          1
 'gaojing': 155,       1
 'yanyiqun': 158,        1
 'shicongli': 100,          1
 'xietangsheng': 149,        1
 'zhaolulu': 148}      1
 